services/metadata/frontend/Button.svelte

```diff
@@ -0,0 +1,21 @@
+<script>
+
+</script>
+
+<button>
+    <slot></slot>
+</button>
+
+<style>
+    button {
+        padding: 0px 12px;
+        cursor: pointer;
+        border-radius: 6px;
+        box-shadow: 1px, 2px, 3px, rgba(0, 0, 0, 0.5);
+        /* background-color: lightgray; */
+    }
+    button:hover {
+        opacity: 0.8;
+    }
+
+</style>
```

Comment:
Change to 2 spaces indentions.

Reply:
actually, this file was an early test and should have been deleted... I'm not using it anywhere.